libs/libvtrutil/src/vtr_memory.h

```diff
@@ -46,6 +47,18 @@ void chunk_delete(T* obj, t_chunk* /*chunk_info*/) {
 //Cross platform wrapper around GNU's malloc_trim()
 // TODO: This is only used in one place within VPR, consider removing it
 int malloc_trim(size_t pad);
+
+inline int memalign(void** ptr_out, size_t align, size_t size) {
+    return posix_memalign(ptr_out, align, size);
+}
+
+// This is a macro because it has to be.  rw and locality must be constants,
+// not just constexpr.
+//
+// This generates a prefetch instruction on all architectures that include it.
+// This is all modern x86 and ARM64 platforms.
```

Comment:
Comment what rw and locality mean, and recommended value.

Reply:
done